examples/control_car/user_config.yaml.example

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+SPEAR:
```

Comment:
These examples should probably change the default map. See `InitializeEngineService`.